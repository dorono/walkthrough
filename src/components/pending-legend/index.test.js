import React from 'react';
import {shallow} from 'enzyme';
import PendingLegend from './index';

describe('PendingLegend', () => {
    it('should render without errors', () => {
        shallow(<PendingLegend />);
    });

    it('should match snapshot', () => {
        expect(shallow(<PendingLegend />)).toMatchSnapshot();
    });
});
