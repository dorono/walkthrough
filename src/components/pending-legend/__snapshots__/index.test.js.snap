// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PendingLegend should match snapshot 1`] = `""`;
